registry/noun/erc7730.json

```diff
@@ -0,0 +1,196 @@
+        ],
+        "required": ["operator", "approved"],
+        "warning": "This will affect ALL your Nouns"
+      },
+      "safeTransferFrom(address,address,uint256)": {
+        "$id": "safeTransferFrom",
+        "intent": "Transfer",
+        "description": "Safely transfer a Noun NFT",
+        "fields": [
+          {
+            "path": "from",
+            "label": "From",
+            "format": "addressName",
+            "params": {
+              "types": ["token"]
+            },
+            "description": "Current owner of the Noun"
+          },
+          {
+            "path": "to",
+            "label": "To",
+            "format": "addressName",
+            "params": {
+              "types": ["token"]
+            },
+            "description": "New owner receiving the Noun"
+          },
+          {
+            "path": "tokenId",
+            "label": "Noun ID",
+            "format": "amount",
+            "description": "The ID of the Noun being transferred"
+          }
+        ],
+        "required": ["from", "to", "tokenId"]
+      },
+      "transferFrom(address,address,uint256)": {
+        "$id": "transferFrom",
+        "intent": "Transfer",
+        "description": "Transfer a Noun NFT",
+        "fields": [
+          {
+            "path": "from",
+            "label": "From",
+            "format": "addressName",
+            "params": {
+              "types": ["token"]
+            },
+            "description": "Current owner of the Noun"
+          },
+          {
+            "path": "to",
+            "label": "To",
+            "format": "addressName",
+            "params": {
+              "types": ["token"]
+            },
+            "description": "New owner receiving the Noun"
+          },
+          {
+            "path": "tokenId",
+            "label": "Noun ID",
+            "format": "amount",
+            "description": "The ID of the Noun being transferred"
+          }
+        ],
+        "required": ["from", "to", "tokenId"]
+      },
+      "delegate(address)": {
+        "$id": "delegate",
+        "intent": "Delegate",
+        "description": "Delegate your Nouns voting power",
+        "fields": [
+          {
+            "path": "delegatee",
+            "label": "Delegate To",
+            "format": "addressName",
+            "params": {
+              "types": ["token"]
+            },
+            "description": "Address to delegate voting power to"
+          }
+        ],
+        "required": ["delegatee"]
+      },
+      "burn(uint256)": {
+        "$id": "burn",
+        "intent": "Burn",
+        "description": "Burn a Noun NFT",
+        "fields": [
+          {
+            "path": "nounId",
+            "label": "Noun ID",
+            "format": "amount",
+            "description": "The ID of the Noun to burn"
+          }
+        ],
+        "required": ["nounId"],
+        "warning": "This action is irreversible"
+      }
+    }
+  },
+  "translations": {
```

Comment:
:wave: this is not supported by the standard, is this an experimentation ?

Reply:
Thanks for catching this! You're right - I'll move the translations directly into the format objects instead of having a separate translations section. Will test on ledger and get back to you.